Sources/Datadog/RUM/Scrubbing/RUMEventsMapper.swift

```diff
@@ -10,13 +10,15 @@ internal typealias RUMViewEventMapper = (RUMViewEvent) -> RUMViewEvent
 internal typealias RUMErrorEventMapper = (RUMErrorEvent) -> RUMErrorEvent?
 internal typealias RUMResourceEventMapper = (RUMResourceEvent) -> RUMResourceEvent?
 internal typealias RUMActionEventMapper = (RUMActionEvent) -> RUMActionEvent?
+internal typealias RUMLongTaskEventMapper = (RUMLongTaskEvent) -> RUMLongTaskEvent?
 
 /// The `EventMapper` for RUM events.
 internal struct RUMEventsMapper {
```

Comment:
Beside `RUMEventsMapperTests`, we also test scrubbing logic in:
* `RUMMonitorTests.testModifyingEventsBeforeTheyGetSend()`
* and `RUMMonitorTests.testDroppingEventsBeforeTheyGetSent()`

Let's update those two with LTs callbacks, so we have their integration coverage.